desktop/php/seniorcare.php

```diff
@@ -317,7 +327,8 @@
 
       <form class="form-horizontal">
         <fieldset>
-          <legend><i class="fas fa-hand-paper"></i> {{Actions arrêt l'avertissement - pour chaque capteur de retour dans les seuils, je dois ?}} <sup><i class="fas fa-question-circle tooltips" title="{{Ces actions seront réalisées lorsqu'un capteur qui était hors seuils retrouve une valeur dans les seuils définis. Pour les messages, vous pouvez utiliser le tag #nom_personne#.}}"></i></sup>
+          <legend><i class="fas fa-hand-paper"></i> {{Actions arrêt d'avertissement - pour chaque capteur de retour dans les seuils, je dois ?}} <sup><i class="fas fa-question-circle tooltips" title="{{Actions réalisées lorsqu'un capteur précédemment hors seuils retrouve ses limites.
```

Comment:
"retrouve ses limites." ca me plaît pas...
"retourne dans ses limites" ?

Reply:
C'est vrai, c'est moche,
2 propositions, si l'une d'entre elles te convient, on peut aussi parler de bornes plutôt que de limites:
- retourne (back to the treshold)
- réintègre

Reply:
Retourne dans ses bornes c'est très bien !